libpromises/eval_context.c

```diff
@@ -206,6 +206,9 @@ struct EvalContext_
         int64_t elapsed;
         Seq *events;
     } profiler;
+
+    EvalContextEvalOrder common;
+    EvalContextEvalOrder agent;
```

Comment:
I don't think these are good variable names - EvalOrder is only in the type but should be in the name in this case. There are potentially many things which can be set both at the common and agent level, not just eval order.

```suggestion
    EvalContextEvalOrder common_eval_order;
    EvalContextEvalOrder agent_eval_order;
```